src/backends/torch/torchdataset.cc

```diff
@@ -533,52 +546,73 @@ namespace dd
                                    inputc->_bw, inputc->width(),
                                    inputc->height());
 
-                // data augmentation can apply here, with OpenCV
-                if (!_test)
+                int samples = _test_crop_samples;
```

Comment:
Actually since we know if we are in test set or not, this could just be
`int samples = _img_rand_aug_cv._crop_params._test_crop_samples`

Reply:
Good point!